packages/stories-block/src/block/icons.js

```diff
@@ -76,3 +81,58 @@ export const CIRCLE_CAROUSEL_CONFIG_ICON = (
 export const GRID_VIEW_CONFIG_ICON = (
   <GridViewConfigIcon width={20} height={27} />
 );
+
+// Defining the SVG like this ensures that IDs are unique
+// even if there are multiple instances of the component.
+// See https://github.com/google/web-stories-wp/issues/8401
+export const BlockIcon = () => {
+  const id = useInstanceId(BlockIcon);
+
```

Comment:
```suggestion
const gradientId = `gradient-${id}`;
const filter          = `filter-${id}`l
```

Seems a little cleaner to setup these as consts up top.